Packs/CybleThreatIntel/Integrations/CybleThreatIntel/CybleThreatIntel.py

```diff
@@ -160,14 +166,21 @@ def get_taxii(self, args: Dict[str, Any]):
                     raise ValueError("Last fetch time retrieval failed.")
 
                 for eachone in content:
-                    save_fetch_time = parser.parse(eachone['timestamp']).replace(tzinfo=pytz.UTC).strftime(
-                        DATETIME_FORMAT)
-
-                taxii_data.append(response)
+                    if eachone.get('confidence'):
+                        current_timestamp = parser.parse(
+                            eachone['confidence']['timestamp']).replace(tzinfo=pytz.UTC).strftime(DATETIME_FORMAT)
+                        if is_first_fetch or datetime.fromisoformat(current_timestamp) > datetime.fromisoformat(save_fetch_time):
+                            save_fetch_time = current_timestamp
+                        else:
+                            skip = True
+
+                if not skip:
+                    taxii_data.append(response)
+
+                    count += 1
+                    if count >= 35:
```

Comment:
why are we using a hard-coded number here?

Reply:
Because I need this number to be fixed, this is the maximum limit. Beyond this limit, the time for polling becomes higher which may lead to timeout.

Reply:
Then let's keep working with the limit param, set a default value of 35, and in the description write that setting the limit to more than that is not recommended.

Reply:
Done